internal/types/eth.go

```diff
@@ -25,9 +25,15 @@ type BlockReplica struct {
 	State           *StateSpecimen
```

Comment:
blockReplica won't have codecVersion?

Reply:
The codec already encodes the version while storing the BSP with something like - 
![Screenshot 2022-07-13 at 14 04 02](https://user-images.githubusercontent.com/26611201/178834342-969edd80-9d2a-4701-8481-8b1e482bbcfc.png)